app/migration_v_0_7.go

```diff
@@ -0,0 +1,20 @@
+package app
```

Comment:
Same comment on migration handler

Reply:
Agreed to do it in a separate PR dedicated to migrations